documentation/modules/operators/con-changing-topic-operator-mode.adoc

```diff
@@ -8,8 +8,7 @@
 [role="_abstract"]
 You can switch between topic management modes when upgrading or downgrading Strimzi, or when using the same version of Strimzi, as long as the mode is supported for that version. 
 
-* To switch from the default bidirectional mode to unidirectional mode, xref:ref-operator-unidirectional-topic-operator-feature-gate-str[enable the `UnidirectionalTopicOperator` feature gate]. 
-* To switch from unidirectional mode to bidirectional mode, check whether finalizers are being used to control topic deletion. If `KafkaTopic` resources are using finalizers, ensure that you do one of the following  before making the switch:
+* To switch from the default unidirectional mode to bidirectional mode you have to xref:ref-operator-unidirectional-topic-operator-feature-gate-str[disable the `UnidirectionalTopicOperator` feature gate], and check whether finalizers are being used to control topic deletion. If `KafkaTopic` resources are using finalizers, ensure that you do one of the following  before making the switch:
```

Comment:
Wouldn't it be better to change the operator config to not use finalisers first, and then disable? Otherwise the users is left with having to run command to remove the finalisers after the downgrade.